### PR TITLE
Added module to arche label in parens

### LINKED ISSUE #25: Archetype circles need full names
CycIC.java - line 541

Somewhere in here remove the name splitting.

### PR DESCRIPTION
**Pull request by gonuke**
Fixes #25 

I chose to add the module name (is this the right term?) in parens after the archetype name.

### CODE REVIEW

File: cyclist/src/edu/utexas/cycic/Cycic.java [hunk 1]
```diff
@@ -536,10 +536,11 @@ public void retrieveSchema(String rawMetadata) {
 	public void buildDnDCircle(FacilityCircle circle, int i, String name){
 		circle.setRadius(40);
 		circle.setStroke(Color.BLACK);
-		circle.setFill(Color.web("#CF5300"));
+        circle.rgbColor=VisFunctions.stringToColor(name);
```
Comment (gidden) [inline cyclist/src/edu/utexas/cycic/Cycic.java]: also indentation issues

**Comment by gidden** [comment]
hi @gonuke, could it be possible to reduce the font on the parens label? Ideally, we would want the out of the box labels to not cause ellipses

image

**Comment by gonuke** [comment]
Mine fits better, but I'll take a look...

**Comment by gonuke** [comment]
I think this comes from combining with #42 - I think I accidentally made the font larger

**Comment by gidden** [comment]
ah gotcha, I'll look without #42

**Comment by gidden** [comment]
Hmm yes, there is still wrapping. Perhaps that is fine? Maybe the font can be set dynamically? (maybe that's too much work...)

What do you think?

image

**Comment by gonuke** [comment]
I can make it a little smaller, but the arbitrary case is hard to manage without some user control - which is probably a bunch of work

**Comment by gidden** [comment]
yes, I agree that the general case may be much harder. is there a global list of known archetypes? if so, would it be hard to only add the subtext if there are multiple entries?

**Comment by gonuke** [comment]
Yes - adding the module string conditionally is a bigger lift!  Probably would need to keep a master list of known archetypes, and define its display name based on its uniqueness.

**Comment by gonuke** [comment]
I'm closing this, as I've pulled it into #42

**Pull request closed without merging**